src/System.Management.Automation/FormatAndOutput/DefaultFormatters/PowerShellCore_format_ps1xml.cs

```diff
@@ -766,14 +766,14 @@ private static IEnumerable<FormatViewDefinition> ViewsOf_System_Management_Autom
                             $maxDepth = 10
                             $ellipsis = ""`u{2026}""
                             $resetColor = ''
-                            if ($Host.UI.SupportsVirtualTerminal -and !(Test-Path env:__SuppressAnsiEscapeSequences)) {
+                            if ($Host.UI.SupportsVirtualTerminal -and ($null -eq $env:__SuppressAnsiEscapeSequences)) {
```

Comment:
I think the `env:` provider will return empty strings rather than null if a value doesn't exist (in the same way, setting an env var to `[string]::Empty`/`""` will remove it in the .NET API for `SetEnvironmentVariable()`).

Probably safest to use `[string]::IsNullOrEmpty()` for these.

Reply:
will update